readthedocs/api/v2/serializers.py

```diff
@@ -127,11 +129,33 @@ class VersionAdminSerializer(VersionSerializer):
 class BuildCommandSerializer(serializers.ModelSerializer):
 
     run_time = serializers.ReadOnlyField()
+    command = serializers.SerializerMethodField()
 
     class Meta:
         model = BuildCommandResult
         exclude = []
 
+    def get_command(self, obj):
+        # HACK: remove unreadable paths from the command outputs when returning it from the API.
+        # We could make this change at build level, but we want to avoid undoable issues from now
+        # and hack a small solution to fix the immediate problem.
+        #
+        # This converts:
+        #   $ /usr/src/app/checkouts/readthedocs.org/user_builds/<container_hash>/<project_slug>/envs/<version_slug>/bin/python
+        #   $ /home/docs/checkouts/readthedocs.org/user_builds/<project_slug>/envs/<version_slug>/bin/python
+        # into
+        #   $ python
+        project_slug = obj.build.version.project.slug
+        version_slug = obj.build.version.slug
+
+        docroot = settings.DOCROOT
+        # Remove Docker hash from DOCROOT when running it locally
+        if settings.RTD_DOCKER_COMPOSE:
+            docroot = re.sub('/[0-9a-z]+/?$', '', settings.DOCROOT, count=1)
+
+        command = re.sub(f'{docroot}/[0-9a-z]+/{project_slug}/envs/{version_slug}(/bin/)?', '', obj.command, count=1)
```

Comment:
I wonder if we want to remove `count=1`, so if we have multiple paths in the command they will be removed? I don't know how often that happens, but seems fine to have this for now, also.